tests/test_util.py

```diff
@@ -6,7 +6,16 @@
 import pytest
 
 from gunicorn import util
+from gunicorn.errors import AppImportError
+import sys
+import os
 
+# Obtain project root path
+project_root = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+# For finding modules to test
+sys.path.insert(0, project_root)
+sys.path.insert(0, project_root + '/examples')
```

Comment:
We are going to need to create some apps for testing sooner or later so I say let's use this PR an opportunity to create a basic WSGI app in `tests/support.py` and use it here. This way we won't have modify `sys.path`.

Reply:
OK, I'll try to do that.

Reply:
@berkerpeksag You mean to copy-paste [the example app function](https://github.com/benoitc/gunicorn/blob/master/examples/test.py) into `test/support.py`?

Reply:
@lulivi yes, that's what I meant.